### PR TITLE
Documentation for gradient API

### PR DESCRIPTION
**Pull request by EugeneElkin**

### CODE REVIEW

File: Capabilities/Gradient.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Gradient property
+The gradient property is special property that cannot be set as usual property. Instead, you need to set special rule for substitution of color picker property (fill type). See the example below.
```
Comment (AviSander) [inline Capabilities/Gradient.md]: The gradient property is a property that cannot be set as standard property. Instead, you need to set a rule for substitution of the color picker property (fill type). See the example below.

File: Capabilities/Gradient.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                "property": "fill",
+                    "selector": [
+                        "Category"
+                    ]
+            }
+        }
+    }
+}
+```
+Please, pay attention to `"fill"` and `"fillRule"` properties. The first is color picker, the second is substitution rule for gradient that will substitute "fill" property `visually` when the rule conditions will be reached.
```
Comment (AviSander) [inline Capabilities/Gradient.md]: attention to **the** `"fill"` and `"fillRule"`

Comment (AviSander) [inline Capabilities/Gradient.md]: The first is **the** color picker, the second is **the** substitution rule for **the** gradient that will substitute **the** "fill" property `visually` when the rule conditions will be **met**.

File: Capabilities/Gradient.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+This link between fill property and substitution rule is set in `"rule"`->`"output"` section of `"fillRule"` property.
```
Comment (AviSander) [inline Capabilities/Gradient.md]: section of **the** `"fillRule"` property.

File: Capabilities/Gradient.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+`"Rule"`->`"InputRole"` sets which data role triggers the rule (condition). In tis example, if data role `"Gradient"` contains data then the rule will be applied for `"fill"` property.
```
Comment (AviSander) [inline Capabilities/Gradient.md]: typo: this

File: Capabilities/Gradient.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+The example of data role that triggers the fill rule you can see below (`the last item`).
```
Comment (AviSander) [inline Capabilities/Gradient.md]: Below you can see an example of the data role that triggers the fill rule

File: Capabilities/Objects.md [hunk 6]
```diff
@@ -70,6 +70,11 @@ export interface StructuralTypeDescriptor {
 }
 ```
 
+## Gradient property
+The gradient property is special property that cannot be set as usual property. Instead, you need to set special rule for substitution of color picker property (fill type).
```
Comment (AviSander) [inline Capabilities/Objects.md]: The gradient property is **a** property that cannot be set as **standard** property. Instead, you need to set **a** rule for substitution of **the** color picker property (fill type).

File: Capabilities/Gradient.md (hunk 1 above)
Comment (EugeneElkin) [reply Capabilities/Gradient.md]: Done!

File: Capabilities/Gradient.md (hunk 2 above)
Comment (EugeneElkin) [reply Capabilities/Gradient.md]: Done!

Comment (EugeneElkin) [reply Capabilities/Gradient.md]: Done!

File: Capabilities/Gradient.md (hunk 3 above)
Comment (EugeneElkin) [reply Capabilities/Gradient.md]: Done!

File: Capabilities/Gradient.md (hunk 4 above)
Comment (EugeneElkin) [reply Capabilities/Gradient.md]: Done!

[the remark this answers is not in the document]
Comment (EugeneElkin) [reply Capabilities/Gradient.md]: Done!

File: Capabilities/Gradient.md (hunk 5 above)
Comment (EugeneElkin) [reply Capabilities/Gradient.md]: Done!

File: Capabilities/Objects.md (hunk 6 above)
Comment (EugeneElkin) [reply Capabilities/Objects.md]: Done!

**Pull request merged**